.github/workflows/build.yml

```diff
@@ -304,37 +210,28 @@ jobs:
 
       - name: "package: cleanup"
         run: |
-          sudo find vendor/dist -mindepth 1 -maxdepth 1 ! -name "vendor-*.tar.gz" -exec rm -rf {} \;
+          sudo find vendor/dist -mindepth 1 -maxdepth 1 ! -name "vendor-Linux.tar.gz" -exec rm -rf {} \;
```

Comment:
Is this the same as 
`rm -rf vendor/dist/vendor-Linux.tar.gz`

Reply:
yes it is 😁  Nice spotting

Reply:
ah, it finds everything _other_ than the `vendor-{OS}.tar.gz` archive and deletes it.